packages/datadog-plugin-couchbase/src/index.js

```diff
@@ -0,0 +1,138 @@
+'use strict'
+
+const Tags = require('../../../ext/tags')
+const Kinds = require('../../../ext/kinds')
+const analyticsSampler = require('../../dd-trace/src/analytics_sampler')
+
+function startQuerySpan (tracer, config, query) {
+  const childOf = tracer.scope().active()
+  const span = tracer.startSpan('couchbase.query', {
+    childOf,
+    tags: {
+      'db.type': 'couchbase',
+      'span.type': 'sql',
+      'component': 'couchbase',
+      'service.name': config.service || `${tracer._service}-couchbase`,
+      'resource.name': query,
+      [Tags.SPAN_KIND]: Kinds.CLIENT
+    }
+  })
+
+  analyticsSampler.sample(span, config.analytics)
+
+  return span
+}
+
+function onRequestFinish (emitter, span) {
+  let rowsListener // eslint-disable-line prefer-const
```

Comment:
Node and eslint were giving me conflicting errors/warnings depending on how I defined this and `errorListener`. But since I'm no longer removing the listeners once a span finishes, I'll just simplify this thing.